sorc/ncep_post.fd/INITPOST_GFS_NETCDF_PARA.f

```diff
@@ -2628,21 +2642,31 @@ subroutine read_netcdf_2d_para(ncid,im,jsta,jsta_2l,jend,jend_2u, &
       real,intent(out)   :: buf(im,jsta_2l:jend_2u)
       integer            :: varid,iret,jj,i,j
       integer            :: start(2), count(2)
+      real,parameter     :: spval_netcdf=9.99e+20
+      real               :: fill_value
+      real,parameter     :: small=1.E-6
```

Comment:
suggest using "small" already defined in params:
use params_mod,   only: small

Reply:
Updated.